app/views/templates/base.php

```diff
@@ -75,6 +75,7 @@
 } ?><?= $title_productname ?></title>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Transvision is a Web application targeting the Mozilla localization community, created and maintained by the French Mozilla community">
```

Comment:
We should be more specific on what it is exactly.
* We don't need to mention that the French community did it
* We need to explain that this Web appllication is a translation memory tool

Ex:
Transvision is a translation memory application for the Mozilla localization community. It indexes translations from Mozilla software and websites in over 80 languages and allows searching for strings for QA purposes, downloading the data in TMX format among other features. It also provides a public json API allowing to retrieve translations and metadata for reuse. 

(can be improved, please check for possible typos too ;) )